src/colord.ts

```diff
@@ -174,6 +174,13 @@ export class Colord {
     if (typeof value === "number") return colord({ h: value, s: hsla.s, l: hsla.l, a: hsla.a });
     return round(hsla.h);
   }
+
+  /**
+   * Checks the color with another instance for equality.
+   */
+  public isEqual(color: Colord): boolean {
```

Comment:
It should accept any kind of input to make it easier to use (not only Colord instance).
Check `isReadable` method as an example.

```js
colord('white').isEqual('#FFF') // true
```

Reply:
My bad, good point!